docs/config/01-configuration-file.md

```diff
@@ -102,6 +102,31 @@ without any problems. Karma does not treat a disconnection as an immediate failu
 If the browser reconnects during that time, everything is fine.
 
 
+## browserConsoleLogOptions
+**Type:** Object
+
+**Default:** `{level: "debug", format: "%b %T: %m", terminal: true}`
+
+**Description:** Configure how the browser console is logged with the following
+properties:
+
+```javascript
+{
+  level:  string=,
```

Comment:
I see. Let's write that in words will be better I feel, as I and probably a lot of other people won't know. Just a sentence above/below saying that they are all optional.